### ISSUE TITLE
Orbital scan won't enable either of its scanners

### ISSUE THREAD

**Issue by DynV** [pioneerspacesim/pioneer#5542]
### Observed behaviour
Around the target (Tethys, Sol), with both orbital scanners† no scan will become enabled (in slate blue at the top) from close to the surface to high orbit.

† I changed using `require 'Game'.player:RemoveEquip(require 'Equipment'.misc.orbitscanner_good), require 'Game'.player:AddEquip(require 'Equipment'.misc.orbitscanner)` in the "field" (without selling then purchasing at a station).
[update below]

### Expected behaviour
1 of the 2 orbital scanner become enabled at some distance from the target.

### Steps to reproduce
1. Load gamesave
2. select Tethys
3. while approaching it (having set autopilot low orbit 1st?) have the Scan Manager open and pay attention to the related scan
4. go near the surface
5. change scanner†
6. move away from the target paying attention to the related scan.

__My pioneer version (and OS):__
20230203-win

The attached file was simply renamed so it would go through the upload extension filter, there was no modification to the file content of my save-file.
[Pioneer-Yoan_Faure-20230203-230213-scanned_Dionne,_Sol.zip](https://github.com/pioneerspacesim/pioneer/files/10717077/Pioneer-Yoan_Faure-20230203-230213-scanned_Dionne._Sol.zip)

**Update 1**
It turns out none of the target of the orbital scan missions from the gamesave attached to this post will enable either of the 2 orbital scanners.

**Comment by arfawellesly** [comment]
Encountered similar problem with orbital scan of Vesta. Scan would not start due to Minimum Altitude warning. Moved further away but scan was not available.

**Comment by impaktor** [comment]
@arfawellesly was the difficutly due to some exotic configuration of the orbitals of that system, or could this be a "random" / unknown issue making the scanner not work?

**Comment by arfawellesly** [comment]
I suspect that it may be due to not being able to enter a high orbit around Vesta – the autopilot could not manage this.

I have encountered another issue also – some missions require about 50% of a scan to be completed – but the scanner ignores this and carries out 100% of a scan and only then does it move the data into the completed scans section.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by impaktor** [comment]
> I suspect that it may be due to not being able to enter a high orbit around Vesta – the autopilot could not manage this

There is no guarantee that the mission can be completed on auto-pilot (actually, no surface scan mission can be, since you need to fly so low).

> I have encountered another issue also – some missions require about 50% of a scan to be completed – but the scanner ignores this and carries out 100% of a scan and only then does it move the data into the completed scans section.

Scanning 50% of the planet is not the same as finishing only 50% of the mission (that would be the same as only having scanned 25%).

**Comment by DynV** [comment]
The workaround was given on your discord #technical-support by AxtelSturnclaw#3882

> @DynV require 'Game'.player.equipSet.slots.sensor.__limit = 3

then you add the missing by buying it or using `require 'Game'.player:RemoveEquip(require 'Equipment'.misc.orbitscanner_good)` or `require 'Game'.player:RemoveEquip(require 'Equipment'.misc.orbitscanner)`. I didn't close the issue as the game isn't fixed.

**Comment by armakunizen** [comment]
I have the same issue with my Linux client, downloaded yesterday.
My mission is to scan 70 Ophiuchi A b. It doesn't start there.

I tried the commands above and it didn't help.

Looking at the file data/libs/Equipment.lua turned out you need a "planetscanner" not an "orbitscanner"

**Comment by impaktor** [comment]
@armakunizen 

> I have the same issue with my Linux client, downloaded yesterday.

From where? Pioneer's site? I.e. february 2023 release?

> Looking at the file data/libs/Equipment.lua turned out you need a "planetscanner" not an "orbitscanner"

So you had a mission to do a planet scan, and you only had an orbit scanner installed? Did it work to do the mission with the correct scanner?

**Comment by armakunizen** [comment]
> From where? Pioneer's site? I.e. february 2023 release?
Yes, from there

> So you had a mission to do a planet scan, and you only had an orbit scanner installed? Did it work to do the mission with the correct scanner?
Yes, it worked with the correct scanner. 

I didn't know that there are different scanner types. I bought one from Earth and took the mission. I played pioneer on german language first and I need to check later, maybe I misunderstood something or the localization is incorrect.

As I did not found any information in the wiki, what should be the difference from an orbital scanner and a planet scanner?
